json/Card/fight-on.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "clan": "crab",
+        "cost": 1,
+        "deck_limit": 3,
+        "element": null,
+        "fate": null,
+        "glory": null,
+        "honor": null,
+        "id": "fight-on",
```

Comment:
I don't agree with adding this card to the database. We don't have all the required information (influence_cost).

Reply:
As you say.

Reply:
You can remove the file and commit